tempodb/encoding/vparquet/parquet_reader.go

```diff
@@ -0,0 +1,86 @@
+package vparquet
```

Comment:
This file name stutters, how about `reader_ats.go` ?

Reply:
Good call, I'm thinking just `readers.go`?